app/cdap/components/GridTable/components/Breadcrumb/styles.ts

```diff
@@ -25,6 +25,12 @@ export const useStyles = makeStyles({
     alignItems: 'center',
     marginRight: '30px',
     marginLeft: '34px',
+    '& .MuiBreadcrumbs-li': {
```

Comment:
we can actually try to combine '& .MuiBreadcrumbs-li' and '& .MuiTypography-body1' into single class '& .MuiBreadcrumbs-li, & .MuiTypography-body1'  plz try this

Reply:
done